src/screens/Settings/index.js

```diff
@@ -83,11 +83,15 @@ const Settings = () => {
             text="Websocket Research"
             onPress={() => navigation.navigate('WebsocketResearchScreen')}
           /> */}
-          <ProfileSettingItem text="Delete Account" onPress={showDeleteAccountAlert} />
-          <ProfileSettingItem text="Logout" onPress={doLogout} />
+          <ProfileSettingItem
+            testID="delete"
+            text="Delete Account"
+            onPress={showDeleteAccountAlert}
+          />
+          <ProfileSettingItem testID="logout" text="Logout" onPress={doLogout} />
         </View>
         <View>
-          <TouchableOpacity onPress={turnOnDebugMode}>
+          <TouchableOpacity testID="debugmode" onPress={turnOnDebugMode}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's a good practice to add `testID` for testing purposes. However, it would be better if the `testID` value is more descriptive and consistent with other `testID`s. Here, "debugmode" could be changed to something like "debugModeButton" to make it clear that it's a button for turning on debug mode.
```diff
- <TouchableOpacity testID="debugmode" onPress={turnOnDebugMode}>
+ <TouchableOpacity testID="debugModeButton" onPress={turnOnDebugMode}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->